polyfill/lib/civildate.mjs

```diff
@@ -13,25 +13,40 @@ export class CivilDate {
     const { year, month, day } = plus({}, { years, months, days });
     this[DATA] = { year, month, day };
   }
+  get [Symbol.toStringTag] () { return 'CivilDate'; }
```

Comment:
these shouldn’t be getters (unless they’re going to brand check before returning the string, which I’d be super in favor of)

Reply:
Why? What?

Since class public fields are not a thing yet, ...
Also what do you mean by "brand check"? Check that it's an instance of ?
Explain, I'm confused.

Reply:
Since class fields aren’t a thing yet, it simply can’t live inside the class body. You’d want to do `CivilDate.prototype[Symbol.toStringTag] = ‘CivilDate’;` after the class definition.

A brand check would be if it threw, or returned null, when the receiver lacked the right internal slot - in other words, if you did `new CivilDate()[Symbol.toStringTag]` you’d get `’CivilDate’`, but if you did `CivilDate.prototype.toStringTag` it would throw.